src/test/java/SalesforceModules/CasesUtil.java

```diff
@@ -0,0 +1,383 @@
+    }
+
+    /**
+     * Delete Case
+     *
+     *
+     *  @param logStep the log
+     */
+    public static void deleteCase(String logStep) throws InterruptedException {
+        selectDeleteButton_case("Selected delete button");
+        clickDeleteButton_case("Clicked delete button");
+
+    }
+
+    /**
+     * click search in case section.
+     *
+     * @param logStep the log
+     */
+    public static void clickSearch_case(String logStep) throws InterruptedException {
+        KeywordUtil.waitForVisible(ServiceSupportObject.searchbutton_case);
+        KeywordUtil.delay(3000);
+        KeywordUtil.click(ServiceSupportObject.searchbutton_case,logStep);
+        KeywordUtil.delay(3000);
+        KeywordUtil.inputText(ServiceSupportObject.searchbutton_case, "00001058", logStep);
+
+    }
+
+    /**
+     * Search Case
+     *
+     *
+     *  @param logStep the log
+     */
+    public static void searchCase(String logStep) throws InterruptedException {
+        KeywordUtil.delay(2000);
+        clickCasesTab("Click on case Tab");
+        clickSearch_case("Clicked search button");
+
+    }
+
+    /**
+     * Enter phone number field for new account.
+     *
+     * @param logStep the log
+     * @param PhoneNumber1 the phoneNumber1
+     */
+    public static void enterPhoneNumber1(String PhoneNumber1, String logStep){
+        KeywordUtil.waitForVisible(ServiceSupportObject.PhoneNumber1);
+        KeywordUtil.inputText(ServiceSupportObject.PhoneNumber1, PhoneNumber1,logStep);
+    }
+
+    /**
+     * Enter account name field for new account.
+     *
+     * @param logStep the log
+     * @param name the account name
+     */
+    public static void enterAccountName(String name, String logStep){
+        KeywordUtil.waitForVisible(ServiceSupportObject.AccountName1);
+        KeywordUtil.inputText(ServiceSupportObject.AccountName1, name,logStep);
+    }
+
+    /**
+     * click save button for case section in search flow
+     *
+     * @param logStep the log
+     */
+    public static void saveBtnOfNewAccount_ss(String logStep){
+        KeywordUtil.waitForVisible(ServiceSupportObject.saveBtnOfNewAccount);
+        KeywordUtil.click(ServiceSupportObject.saveBtnOfNewAccount,logStep);
+
+    }
+
+    /**
+     * click save button for case section in search flow
+     *
+     * @param logStep the log
+     */
+    public static void saveBtnOfNewCase_ss(String logStep){
+        KeywordUtil.waitForVisible(ServiceSupportObject.saveBtnOfNewCase);
+        KeywordUtil.click(ServiceSupportObject.saveBtnOfNewCase,logStep);
+    }
+
+    /**
+     * Enter account name field for new account.
+     *
+     * @param logStep the log
+     * @param AccountName1 the account name
+     * @param PhoneNumber1 the PhoneNumber1
+     */
+    public static void newCaseByCreatingNewAccount(String AccountName1, String PhoneNumber1,String origin,String logStep) throws InterruptedException {
```

Comment:
Change arguments name from AccountName1 to accountName and PhoneNumber1 to phoneNumber, it should start from small letter